Whats New for March 2022 - App Store and Quality Score for Product Mo… (#1117)
akeneo/pim-helpcenter: content/updates/2022-03/01-AppStore.md

---
pim_announcement_img: ./img/AkeneoAppStore.png
pim_announcement_alt_img: Akeneo App Store
pim_announcement_audience:
- EE
- GE
---

# Akeneo App Store
::: meta-data type="New" features="Connectivity" available="mid-March" in="EE,GE" link-to-doc="../articles/how-to-connect-my-pim-with-apps.html"

> Introducing the next generation of connectivity for everyone working with Akeneo PIM: Apps and Akeneo App Store. Right from the Connect tab you will now see the App Store where you can discover, browse and connect with trusted Apps that connect to your technology stack and can extend Akeneo PIM in exciting new ways with just a few clicks. There are several new Apps for you to try today, with more to come. You can also access, manage and configure your installed Apps from the Connected Apps screen. A whole new world of connectivity has just opened up to you.

Reviewer: ```suggestion
Introducing the next generation of connectivity with Akeneo PIM: Apps and Akeneo App Store. Right from the Connect Navigation, you will now see the App Store where you can discover, browse and connect with trusted Apps. Connect your existing technology stack or extend Akeneo PIM in exciting new ways with just a few clicks. There are several new Apps for you to try today, with more to come. In the Connected Apps screen, you can also access, manage and configure your installed Apps. A whole new world of connectivity has just opened up to you.
```
